modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -125,7 +130,7 @@ To test your app:
 . Click the *Status* column for the `osv2-demo` app to display its details.
 . Copy the *App url* value:
 +
-image::os-tutorial-rtm-deploy-url.png["App url in Runtime Manager"]
+image::os-tutorial-rtm-deploy-url.png["Runtime Manager app details pane showing the App url"]
```

Comment:
```suggestion
image::os-tutorial-rtm-deploy-url.png["Runtime Manager app details pane showing the App URL"]
```

Reply:
The label on the UI element is "Url". For better or for worse.